hayeon/programmers-0/평행.js

```diff
@@ -0,0 +1,24 @@
+function solution(dots) {
+  let inclinationArr = [];
+  for (let i = 0; i < dots.length; i++) {
+    for (let j = i + 1; j < dots.length; j++) {
+      inclinationArr.push(
+        (dots[i][1] - dots[j][1]) / (dots[i][0] - dots[j][0])
+      );
+    }
+  }
+  let setArr = new Set();
+
+  for (let el of inclinationArr) {
+    setArr.add(el);
+  }
+
+  return [...setArr].length === inclinationArr.length ? 0 : 1;
+}
```

Comment:
```suggestion
function solution(dots) {
  const inclinationArr = [];
  for (let i = 0; i < dots.length; i++) {
    for (let j = i + 1; j < dots.length; j++) {
      inclinationArr.push(
        (dots[i][1] - dots[j][1]) / (dots[i][0] - dots[j][0])
      );
    }
  }
  const inclinationSet = new Set();

  for (let el of inclinationArr) {
    inclinationSet.add(el);
  }

  return [...inclinationSet].length === inclinationArr.length ? 0 : 1;
}
```
- 기울기를 이용한 풀이 잘봤습니다🙇‍♂️ 단, inclinationArr, setArr 모두 let으로 선언 할 이유가 없어 보이는데 `const`로 선언해주시길 바랍니다.
- setArr도 차라리 `inclinationSet`이 적절한 변수명같습니다. setArr이라는데 배열인지 셋인지 구분이 안가네요 🥲
- 윤우님은 기울기가 아닌 각도를 이용해서 푸셨기 때문에 윤우님 코드도 참고해주시길 바랍니다!!
- 마지막으로 좀 더 안정적이고 견고한 코드를 짜기 위해 let, const 선언 좀 더 신경써주시길 바랍니다 ~~!!

Reply:
넵넵 신경써서 선언하겠숩니닷..!! 역시 변수 이름 짓는게 제일 힘드네요ㅠㅠ 윤우님 코드도 참고해서 보겠습니다!